Source/JavaScriptCore/runtime/AbstractModuleRecord.cpp

```diff
@@ -764,11 +764,11 @@ JSModuleNamespaceObject* AbstractModuleRecord::getModuleNamespace(JSGlobalObject
         RETURN_IF_EXCEPTION(scope, nullptr);
         switch (resolution.type) {
         case Resolution::Type::NotFound:
-            throwSyntaxError(globalObject, scope, makeString("Exported binding name '", String(name.get()), "' is not found."));
+            throwSyntaxError(globalObject, scope, makeString("Exported binding name '"_s, String(name.get()), "' is not found."_s));
```

Comment:
Should be able to pass `*name` or `name.get()` without calling `String` and avoid the reference count churn.